e2e-tests/constants/routes/insurance/exporter-business.js

```diff
@@ -8,6 +8,7 @@ const CHECK_YOUR_ANSWERS = 'check-your-answers';
 export const EXPORTER_BUSINESS = {
   ROOT: EXPORTER_BUSINESS_ROOT,
   COMPANY_DETAILS: `${EXPORTER_BUSINESS_ROOT}/${COMPANY_DETAILS_ROOT}`,
+  COMPANIES_HOUSE_DOWN: `${EXPORTER_BUSINESS_ROOT}/company-details-unavailable`,
```

Comment:
Can we rename all instances of `COMPANIES_HOUSE_DOWN` to `COMPANIES_HOUSE_UNAVAILABLE` so it's aligned with the route?